docs/en/observability/monitor-k8s/monitor-k8s-application-performance.asciidoc

```diff
@@ -208,11 +208,11 @@ kubectl apply -f demo.yml
 ----
 
 [discrete]
-=== View your traces in {kib}
+=== View your application's traces in {kib}
 
-To view your application's trace data, open {kib} and go to *{observability} → Service inventory*.
+To open **Service Inventory**, find **Applications** in the main menu or use the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
```

Comment:
Starting the section with `To open Service Inventory` when the title doesn't mention it feels a bit weirt.... what about?

```suggestion
Applications' trace data is available in the **Service Inventory**.
To open **Service Inventory**, find **Applications** in the main menu or use the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
```

With or without newline between the sentences, whatever you like the most.

Reply:
good point!